apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -67,6 +68,9 @@ class SectionAssessments extends Component {
         <div>
           <MultipleChoiceByStudentContainer />
         </div>
+        <div>
```

Comment:
are the `div`s around `MultipleChoiceByStudentContainer` and `StudentsMCSummaryContainer` needed?

Reply:
I removed the divs and there is not change.